File: backend/src/main/java/com/vtb/payandsave/request/card/CardSettingsRequest.java
package com.vtb.payandsave.request.card;

import com.vtb.payandsave.model.card.enums.CardRoundingStep;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
@Data
public class CardSettingsRequest {
    private boolean active;
    private CardRoundingStep roundingStep;
}

File: frontend/src/app/request/CardSettingsRequest.ts
export class CardSettingsRequest {
  active: boolean;
  roundingStep: CardRoundingStep;

  constructor(active: boolean, roundingStep: CardRoundingStep) {
    this.active = active;
    this.roundingStep = roundingStep;
  }
}

enum CardRoundingStep {
  STEP10, STEP50,STEP100, STEP1000
}
